packages/core/drizzle/0095_handy_skreet.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "latitude"."llm_as_judge_evaluation_metadatas" DROP COLUMN IF EXISTS "configuration";
```

Comment:
I'll do the stupid question here. But the data migration already put the context of this in the new place right?

Reply:
Yes! The code in production is not even aware this attribute exists, everything is being added and read from the new attributes/tables/relations